pandas/tests/exchange/test_utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas.core.exchange.utils import dtype_to_arrow_c_fmt
+
+# TODO: use ArrowSchema to get reference C-string.
+# At the time, there is no way to access ArrowSchema holding a type format string
+# from python. The only way to access it is to export the structure to a C-pointer,
+# see DataType._export_to_c() method defined in
+# https://github.com/apache/arrow/blob/master/python/pyarrow/types.pxi
+
+
+@pytest.mark.parametrize(
+    "pandas_dtype, c_string",
```

Comment:
What about categorical dtype?

Reply:
This function does not support conversion of categoricals - they're handled up in the stack.

Reply:
The first check in the function is about categorical dtype ([link](https://github.com/pandas-dev/pandas/pull/46141/files#diff-3ac5354f7d5725a4944334ae0529ba65f4580e26469dc8626e27172a1513a269R72)).

Reply:
whoops, missed that, thanks!